composer.json

```diff
@@ -17,16 +17,16 @@
     ],
     "require": {
         "php": "^7.1",
-        "symfony/doctrine-bridge": "^2.8 || ^3.1",
         "doctrine/orm": "^2.2",
-        "stephpy/timeline-bundle": "^2.3",
-        "sonata-project/core-bundle": "^3.0",
         "sonata-project/block-bundle": "^3.2",
+        "sonata-project/core-bundle": "^3.0",
+        "sonata-project/easy-extends-bundle": "^2.2",
         "sonata-project/intl-bundle": "^2.2",
-        "sonata-project/easy-extends-bundle": "^2.2"
+        "stephpy/timeline-bundle": "^2.3",
+        "symfony/doctrine-bridge": "^2.8 || ^3.1"
     },
     "require-dev": {
-        "matthiasnoback/symfony-dependency-injection-test": "^0.7.6 || ^1.0",
+        "matthiasnoback/symfony-dependency-injection-test": "^1.0",
```

Comment:
This does not look like sorting to me :P

Reply:
Woops, on the wrong commit...

Reply:
Fixed